jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/annotations/LinterRule.kt

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.cloudformation.annotations
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
We should configure the `ObjectMapper` to ignore unknown properties - rather than doing it for every actual type.